Mage.Sets/src/mage/cards/c/CaseOfTheGatewayExpress.java

```diff
@@ -0,0 +1,148 @@
+ * To solve -- Three or more creatures attacked this turn.
+ * Solved -- Creatures you control get +1/+0.
+ *
+ * @author DominionSpy
+ */
+public final class CaseOfTheGatewayExpress extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("choose target creature you don't control.");
+
+    static {
+        filter.add(TargetController.NOT_YOU.getControllerPredicate());
+    }
+
+    public CaseOfTheGatewayExpress(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{W}");
+
+        this.subtype.add(SubType.CASE);
+
+        // When this case enters the battlefield, choose target creature you don't control. Each creature you control deals 1 damage to that creature.
+        Ability initialAbility = new EntersBattlefieldTriggeredAbility(
+                new CaseOfTheGatewayExpressEffect());
+        initialAbility.addTarget(new TargetPermanent(StaticFilters.FILTER_CREATURE_YOU_DONT_CONTROL));
+        // To solve -- Three or more creatures attacked this turn.
+        Condition toSolveCondition = new CaseOfTheGatewayExpressCondition();
+        // Solved -- Creatures you control get +1/+0.
+        Ability solvedAbility = new SimpleStaticAbility(new ConditionalContinuousEffect(
+                new BoostControlledEffect(1, 0, Duration.WhileOnBattlefield),
+                SolvedSourceCondition.SOLVED, ""));
+
+        this.addAbility(new CaseAbility(initialAbility, toSolveCondition, solvedAbility)
+                .addHint(new CaseOfTheGatewayExpressHint(toSolveCondition)),
+                new AttackedThisTurnWatcher());
+    }
+
+    private CaseOfTheGatewayExpress(final CaseOfTheGatewayExpress card) {
+        super(card);
+    }
+
+    @Override
+    public CaseOfTheGatewayExpress copy() {
+        return new CaseOfTheGatewayExpress(this);
+    }
+}
+
+class CaseOfTheGatewayExpressEffect extends OneShotEffect {
+
+    CaseOfTheGatewayExpressEffect() {
+        super(Outcome.Damage);
+        staticText = "choose target creature you don't control. " +
+                "Each creature you control deals 1 damage to that creature.";
+    }
+
+    private CaseOfTheGatewayExpressEffect(final CaseOfTheGatewayExpressEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public CaseOfTheGatewayExpressEffect copy() {
+        return new CaseOfTheGatewayExpressEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Permanent permanent = game.getPermanent(source.getFirstTarget());
+        if (permanent == null) {
+            return false;
+        }
+        for (Permanent creature : game.getBattlefield().getActivePermanents(
+                StaticFilters.FILTER_CONTROLLED_CREATURE,
+                source.getControllerId(), source, game)) {
+            if (creature == null || creature.getPower().getValue() < 1) {
```

Comment:
assuming this was copied from something that cared about the creature's power, but not relevant here so needs removed